core/src/main/java/org/openedx/core/ui/ComposeCommon.kt

```diff
@@ -165,7 +165,7 @@ fun Toolbar(
             modifier = Modifier
                 .testTag("txt_toolbar_title")
                 .align(Alignment.CenterVertically)
-                .padding(end = 16.dp),
+                .padding(start = if (canShowBackBtn.not()) 16.dp else 0.dp, end = 16.dp),
```

Comment:
nit: it's better to use positive logic: if (canShowBackBtn) 0.dp else 16.dp

but you can skip this comment